client/src/stylesheets/components/_HeroCTA.scss

```diff
@@ -0,0 +1,13 @@
+@import '../utils/placeholders';
+
+.hero_wrapper {
+  background-color: seagreen;
+  height: 33vh;
+  padding: 1em;
+}
+
+.open_map_button {
+  @extend %btn_primary;
+  padding: 1em;
+  height: 4em;
```

Comment:
I know we did this for quick progress, but at some point we should decide whether to go with `em`, `rem`, or `px` for layout things, and then be consistent. The tricky thing with `em` is that it'll change based on the inherited value for `1em`, which might not be the same throughout the app. We should probably use `rem` ("root em", aka 16px) for things that aren't intended to be tied to a component's font size: https://zellwk.com/blog/rem-vs-em/